lib/sql-string.js

```diff
@@ -61,6 +61,10 @@ SqlString.escape = function(val, stringifyObjects, timeZone, dialect, field) {
     return val + '';
   }
 
+  if (field && field.type && field.type.validate) {
+    field.type.validate(val);
```

Comment:
What is the best place to put this without losing the type validation when performing queries?